docs/amber_script.md

```diff
@@ -75,31 +80,33 @@ TODO(dneto): Support half-precision floating point.
 Sized arrays and structures are not currently representable.
 
 ```
-// Filling the buffer with a given set of data. The values must provide
-// <size_in_bytes> of <type> data. The data can be provided as the type or
-// as a hex value.
-
+// Filling the buffer with a given set of data. The values must be
+// of <type> data. The data can be provided as the type or as a hex value.
 BUFFER <name> DATA_TYPE <type> DATA
 <value>+
 END
 
+// Defines a buffer which is filled with data as specified by the `initializer`.
 BUFFER <name> DATA_TYPE <type> SIZE <size_in_items> <initializer>
+
+// Creates a buffer which will store the given `FORMAT` of data. These
+// buffers are used as image and depth buffers in the `PIPELINE` commands.
+// The buffer will be sized based on the `RENDER_SIZE` of the `PIPELINE`.
+BUFFER <name> FORMAT <format_string>
 ```
 
 TODO(dsinclair): Does framebuffer need a format attached to it?
 
 #### Buffer Initializers
-Fill the buffer with a single value.
 
 ```
+// Fill the buffer with a single value.
 FILL <value>
-```
-
-Fill the buffer with an increasing value from \<start> increasing by \<inc>.
-Floating point data uses floating point addition to generate increasting values.
-Likewise, integer data uses integer addition to generate increasing values.
 
-```
+// Fill the buffer with an increasing value from \<start> increasing by \<inc>.
+// Floating point data uses floating point addition to generate increasting
+// values. Likewise, integer data uses integer addition to generate increasing
+// values.
```

Comment:
Right now, it doesn't because I wasn't sure how to make it make sense. SERIES_FROM 0 INC_BY 2 for a vec2, would you do <0, 2>, <4, 8> or <0, 0>, <2, 2>.

We do FILL but not SERIES_FROM until we can figure out semantics.